src/randgraphs.jl

```diff
@@ -28,7 +28,10 @@ function DiGraph(nv::Integer, ne::Integer; seed::Int = -1)
     return g
 end
 
-"""Creates an [Erdős–Rényi](http://en.wikipedia.org/wiki/Erdős–Rényi_model)
+doc"""
+erdos_renyi(n::Integer, p::Real; is_directed=false, seed::Integer=-1)
+
```

Comment:
We don't need `doc` preceding docstrings, and I think the function signatures are included in the autogenerated stuff. Can you double check this please?